src/components/BottomNavigation/buttonNavigation.css

```
@@ -0,0 +1,54 @@
+
+.nav-item:hover {
+  color: var(--brand-primary);
+}
+
+.nav-item.payment {
+  transition: box-shadow 0.2s ease;
+  cursor: pointer;
+}
+.nav-item.payment:hover {
+  color: var(--bg-white);
+  box-shadow: inset 0 4px 8px rgba(0, 0, 0, 0.2);
```

Stylelint (17.2.0)

[error] Expected "0.2" to be "20%" (alpha-value-notation)

(alpha-value-notation)